pkg/skaffold/instrumentation/export.go

```diff
@@ -167,6 +167,8 @@ func createMetrics(ctx context.Context, meter skaffoldMeter) {
 	m := global.Meter("skaffold")
 
 	// cloud monitoring only supports string type labels
+	// cloud monitoring only supports 10 labels per metric descriptor
+	// be careful when appending new values to this `labels` slice
```

Comment:
do we track how many labels we are using so far and what they're being used for?

Reply:
I meant this as a comment for the immediate `labels` slice on line 172

Reply:
but you're right, we don't have any document for the exact set across all metrics